src/flow/src/transform/expr.rs

```diff
@@ -351,7 +361,13 @@ impl TypedExpr {
                 Ok(TypedExpr::new(ret_expr, ret_type))
             }
             _var => {
-                if VariadicFunc::is_valid_func_name(fn_name) {
+                if fn_name == "tumble_start" || fn_name == "tumble_end" {
```

Comment:
I see these literals occur many times. Maybe defining them as constants would be better?